float32[6] q
float32[6] speed # 1-80
float32[6] acc	 # 1-40 (not implemented yet)
---
